templates/firefox-ios.xliff

```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
+        <source>Terms of use</source>
+        <note>Show Firefox Browser Terms of Use page from the Privacy section in the Shopping Experience Opt In onboarding Card (Fakespot). See https://www.mozilla.org/privacy/firefox/</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReliabilityScore.Grade.A11y.Label.v120" xml:space="preserve">
+        <source>Grade %@</source>
+        <note>Accessibility label for the Grade labels used in 'How we determine review quality' card and 'How reliable are these reviews' card displayed in the shopping review quality bottom sheet. The placeholder will be replaced by a grade letter (e.g. A).</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQuality.ReliabilityCardTitle.v120" xml:space="preserve">
+        <source>How reliable are these reviews?</source>
+        <note>Title of the reliability card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQuality.ReliabilityRating.AB.Description.v120" xml:space="preserve">
+        <source>Reliable reviews</source>
+        <note>Description of the reliability ratings for rating 'A' and 'B' displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQuality.ReliabilityRating.C.Description.v120" xml:space="preserve">
+        <source>Mix of reliable and unreliable reviews</source>
+        <note>Description of the reliability rating 'C' displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQuality.ReliabilityRating.DF.Description.v120" xml:space="preserve">
+        <source>Unreliable reviews</source>
+        <note>Description of the reliability ratings for rating 'D' and 'F' displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.AdjustedRating.Label.v120" xml:space="preserve">
+        <source>The *adjusted rating* is based only on reviews we believe to be reliable.</source>
+        <note>Adjusted rating label from How we determine review quality card displayed in the shopping review quality bottom sheet. The *text inside asterisks* denotes part of the string to bold, please leave the text inside the '*' so that it is bolded correctly.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.Collapsed.AccessibilityLabel.v120" xml:space="preserve">
+        <source>How we determine review quality Card Collapsed</source>
+        <note>Accessibility label for the up chevron, from 'How we determine review quality' card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.Expanded.AccessibilityLabel.v120" xml:space="preserve">
+        <source>How we determine review quality Card Expanded</source>
```

Comment:
same comment as above

Reply:
Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940